src/ontology/cl-edit.owl

```diff
@@ -32258,6 +32260,18 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005385))
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011148))
 
+# Class: obo:CL_4047001 (liver-resident natural killer cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26330348") Annotation(oboInOwl:hasDbXref "PMID:26639736") Annotation(oboInOwl:hasDbXref "PMID:27798170") Annotation(oboInOwl:hasDbXref "PMID:32351704") Annotation(oboInOwl:hasDbXref "PMID:34230995") Annotation(oboInOwl:hasDbXref "PMID:35865550") obo:IAO_0000115 obo:CL_4047001 "A natural killer cell resident to the liver, in humans this cell type is distinguished from circulating natural killer cells by CD49a or CD69 gene expression. Liver-resident natural killer cells have also been shown to express CCR5, EOMES, KLRB1, GZMK, and CXCR6 in humans.")
+AnnotationAssertion(terms:contributor obo:CL_4047001 <https://orcid.org/0009-0005-7919-4905>)
```

Comment:
Add as **related_synonym:**
liver-specific natural killer cell 
PMID: 26330348

the definition states  `Liver-resident natural killer cells` but the cell label is `liver resident natural killer cell` remove `-` and add `liver-resident natural killer cell` as a related synonym

optional: consider adding a `comment`  section for the role of liver resident natural killer cells during inflammation. You can consider looking at this reference PMID:34230995

Reply:
added synonym and com ent. Did not remove '- ' in description as cell label is 'liver-resident natural killer cell', but did add 'liver resident natural killer cell' as synonym